src/Screens/Cart/Cart.js

```diff
@@ -0,0 +1,223 @@
+import "./cart.css";
+import Card from "./../../Components/Card/Card";
+import { useEffect } from "react";
+import { useProducts } from "./../../Context/products-context";
+import { Link } from "react-router-dom";
+import Empty from "./../../Components/Empty/Empty";
+import emptyImage from "./../../Media/empty-cart.png";
```

Comment:
hey, this thing can be improved to absolute addresses.
it needed a little configuration but it is defiantly worth doing. 
You can read about it or else let me know when you are free I'll help you do it. Its just a 10 min thing. 
For now these imports are good but you should update them down the line.